### ISSUE TITLE
Hyperlinks do not work at W3C documents

### ISSUE THREAD

**Issue by koppor** [wkhtmltopdf/wkhtmltopdf#2618]
I downloaded the "Bleeding Edge" version as of today and tested it with 

```
"c:\Program Files\wkhtmltopdf\bin\wkhtmltopdf" "http://www.w3.org/TR/DOM-Parsing/" test.pdf
```

at Windows 10. When I open the resulted PDF, the hyperlinks of the TOC are not clickable. Links starting with `http://` are clickable, but I assume that Acrobat Reader does some magic here...

**Comment by koppor** [comment]
There is a similar issue in pandoc (https://github.com/jgm/pandoc/issues/2438). The developers identified that `<section>` elements cannot be used as targets currently. Maybe it's the same issue here?

**Comment by Tomsgu** [comment]
Closing as it seems to be resolved in 0.12.5. Can't reproduce the issue.
